packages/sleepy-launcher-bin/sleepy-launcher-bin.pacscript

```diff
@@ -0,0 +1,44 @@
+pkgname="sleepy-launcher-bin"
+gives="sleepy-launcher"
+pkgver="1.0.0"
+pkgdesc="Sleepy game launcher for Linux with telemetry disabling"
+repology=("project: sleepy-launcher")
+source=(
+  "https://github.com/an-anime-team/sleepy-launcher/releases/download/${pkgver}/sleepy-launcher"
+  "icon.png::https://raw.githubusercontent.com/an-anime-team/sleepy-launcher/main/assets/images/icon.png"
+)
+sha256sums=(
+  "ebb736f2a324ff9ef657248eadc0f329efb850dc72be4cb6664d86b139916983"
+  "SKIP"
+)
+depends=("xdelta3" "p7zip-full" "webp")
+maintainer=("vbrabandt2005 <vbrabandt@proton.me>")
+incompatible=("ubuntu:bionic" "ubuntu:focal" "ubuntu:jammy" "ubuntu:kinetic" "debian:stretch" "debian:buster" "debian:bullseye")
+breaks=("${gives}" "${gives}-deb" "${gives}-app" "${gives}-git")
+
+optdepends=(
+  "mangohud: FPS Hud/GUI"
+  "gamemode: Game Optimizations"
+)
+
+package() {
+  cd "${_archive}"
```

Comment:
Don't use _archive.

Reply:
Sorry for being kinda lost, but I should just remove the `cd "${_archive}"` line entirely right? I was looking at the reference in the wiki and other people's commits, I see `${gives}` or `${pkgname}-${pkgver}` being used but for me I keep getting `/tmp/pacstall/bwrapenv.zOXikXkHJB: line 228: cd: sleepy-launcher (or -bin): Not a directory`

I saw PR #6140 which doesn't cd to anywhere and I assume mine should also be correct now

Reply:
So, `${_archive}` was added as a measure to make the jump from pre-5.0.0 to 5.0.0 since we could add multiple items to download, instead of just having `url` which took one variable. It is deprecated for all new packages and we're in the process of removing it entirely.

Reply:
Oh that's good to know, but the script is ok now right? so I could set it for being ready to review

and in the future I will have to remove `${_archive}` for the other packages I maintain too correct?

Reply:
Yeah you can set it for review, and yes, you should remove it from all packages you maintain when you have time. We will remove them for 6.0.0, but we'd prefer not to do *all* the work :)

Reply:
Nice, thanks for the info :)